### PR TITLE
Added support for task scheduling

### PR DESCRIPTION
**Pull request by PurtucRobert**
- Added the function that cleans the unconfirmed reservations 
 - Added the function that cleans the playing time that already happened

### CODE REVIEW

File: cinemy/cinema/tasks/tasks.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def clean_unconfirmed_reservations():
+    """
+    This function will clean all the unconfirmed reservations
+    at 30 minutes prior to starting the movie
+    """
+    time_now = timezone.now()
+    playing_times = PlayingTime.objects.annotate(
+        time_min=(time_now - F("start_time"))
+    ).filter(time_min__lte=timedelta(minutes=30))
+    for playing_time in playing_times:
```
Comment (ghost) [inline cinemy/cinema/tasks/tasks.py:16]: I wrote on the group, modified the ticket. Please modify this, we only need to mark them as expired, not delete them.

**Review by drejleeo — CHANGES_REQUESTED** [review]

File: cinemy/cinema/tasks/tasks.py (hunk 1 above)
Comment (PurtucRobert) [reply cinemy/cinema/tasks/tasks.py:16]: Done

**Review by maxinasa — COMMENTED** [review]
LGTM

**Comment by PurtucRobert** [comment]
> LGTM

If everything is ok, approve it.

**Review by drejleeo — APPROVED** [review]

**Pull request merged**